packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItems/DataSeriesFormSettings.jsx

```diff
@@ -118,6 +120,40 @@ const DataSeriesFormSettings = ({ cardConfig, onChange, i18n }) => {
           value={content?.decimalPrecision}
         />
       </div>
+      <div className={`${baseClassName}--input--toggle-field ${baseClassName}--input`}>
+        <span>{mergedI18n.includeZeroOnXaxis}</span>
+        <ToggleSmall
+          data-testid="includeZeroOnXaxis toggle"
```

Comment:
did we mean to have a space in the test-id?

Reply:
good point I'll remove